code/espurna/settings.ino

```diff
@@ -213,7 +213,12 @@ void _settingsInitCommands() {
 
     settingsRegisterCommand(F("GPIO"), [](Embedis* e) {
         if (e->argc < 2) {
-            DEBUG_MSG_P(PSTR("-ERROR: Wrong arguments\n"));
+            DEBUG_MSG_P(PSTR("Printing all GPIO pins:\n"));
```

Comment:
This makes sense in general, but maybe save it for later to show used gpios instead?

Reply:
Actually the PR was for 2 things at once ;) But yes I understand your concern, it's better to split them. I have updated the PR now.